lib/chef/chef_fs/file_system/chef_server/cookbooks_dir.rb

```diff
@@ -29,6 +29,15 @@ class Chef
   module ChefFS
     module FileSystem
       module ChefServer
+        # cookbooks/
+        #   apache2/
+        #   mysql/
+        # cookbook_artifacts/
+        #   apache2/
+        #     1.0.0/
+        #     1.0.1/
+        #   mysql/
+        #     2.0.5/
```

Comment:
For clarity, the `cookbook_artifacts` should probably have identifiers like the hash strings they'll be in practice.